__tests__/rpcProvider.test.ts

```diff
@@ -99,6 +105,29 @@ describeIfRpc('RPCProvider', () => {
       expect(transaction).toHaveProperty('transaction_hash');
     });
 
+    test('getInvokeEstimateFee', async () => {
+      const fetchSpy = jest.spyOn(rpcProvider as any, 'fetchEndpoint');
+
+      const randomWallet = stark.randomAddress();
+      const invocation: Invocation = {
+        contractAddress: randomWallet,
+        entrypoint: 'transfer',
+        calldata: [randomWallet, '10', '0'],
+        signature: [],
+      }
```

Comment:
Multiple linting issues

```suggestion
      };
```